exist-core/src/main/java/org/exist/xquery/XPathUtil.java

```diff
@@ -77,27 +78,26 @@ public final static Sequence javaObjectToXPath(Object obj, XQueryContext context
         } else if (obj instanceof Sequence) {
             return (Sequence) obj;
         } else if (obj instanceof String) {
-            final StringValue v = new StringValue((String) obj);
+            final StringValue v = new StringValue(null, (String) obj);
```

Comment:
is this really needed this way? cant we use the single argument constructor here?

Reply:
@dizzzz: I re-added all the expressionless value constructors.